### PR TITLE
[ETP-395] Add entry point for IAC support

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ETP-395]

Dependency for: https://github.com/moderntribe/event-tickets-plus/pull/1062

No artifacts are available for this. Check the PR linked above for details.

[ETP-395]: https://moderntribe.atlassian.net/browse/ETP-395

### CODE REVIEW

File: src/admin-views/attendees.php [hunk 1]
```diff
@@ -110,7 +110,11 @@
 							<li>
 								<strong><?php echo esc_html( $ticket_name ) ?>:&nbsp;</strong><?php
 								echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
-								?></li>
+
+								$this->set( 'ticket_item_for_overview', $ticket );
+								$this->do_entry_point( 'overview_section_after_ticket_name' );
```
Comment (sc0ttkclark) [inline src/admin-views/attendees.php:115]: Let's add a phpdoc block like we do with normal actions, so a `@version` and description

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs phpdoc for entry point

File: src/admin-views/attendees.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/admin-views/attendees.php:115]: `@since` I meant

Comment (rafsuntaskin) [reply src/admin-views/attendees.php:120]: Thanks. Updated.

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**